modules/data.land/R/dataone_download.R

```diff
@@ -24,23 +25,16 @@ dataone_download = function(id, username, CNode = "PROD", lazyLoad = FALSE, quie
   pkg <- dataone::getDataPackage(d1c, id = id, lazyLoad = lazyLoad, quiet = quiet, limit = "1MB") # what is the standard limit for pecan downloads?
   files <- datapack::getValue(pkg, name="sysmeta@formatId")
   n <- length(files) # number of files
-  
-  # fileath to /dbfiles 
-  fp <- "/fs/data1/pecan.data/dbfiles/"
-  
+
   # make new directory within this directory
-  newdir <- paste(fp, "NewData_", username, sep = "")
-  system(paste("mkdir", newdir))
-  
-  # switch to new directory -- unsure if I should do this in R or in unix
-  # system(paste("cd", fp, sep = " ")) 
-  setwd(newdir)
+  newdir <- paste(filepath, "NewData_", username, sep = "")
```

Comment:
Ok, sounds good. Will there be any issues with the creation of the newdir if the id contains '/' in it? I ran into issues with this before... (e.g. id = doi:10.6073/pasta/63ad7159306bc031520f09b2faefcf87)